src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -568,11 +568,11 @@ impl LocalHummockStorage {
     }
 }
 
-pub type StagingDataIterator = OrderedMergeIteratorInner<
+pub type StagingDataIterator = MergeIterator<
     HummockIteratorUnion<Forward, SharedBufferBatchIterator<Forward>, SstableIterator>,
 >;
 pub type HummockStorageIteratorPayloadInner<'a> = SkipWatermarkIterator<
-    UnorderedMergeIteratorInner<
+    MergeIterator<
```

Comment:
In the future maybe we can consider moving the iterators in `StagingDataIterator` to here to merge together instead of having two layers of merge iter.